scripts/check_python

```diff
@@ -11,8 +11,10 @@ if hash "$PYTHON" 2>/dev/null; then
 	# Use full path from now on as it makes messages clearer
 	PYTHON=$(which $PYTHON)
 	PYTHON_VERSION=$($PYTHON -c 'import sys; print("%d.%d" % (sys.version_info[0], sys.version_info[1]))')
-	if [[ "$PYTHON_VERSION" != "2.7" ]]; then
-		echo "check-python: Python v2.7 not found (version reported by '$PYTHON' is $PYTHON_VERSION)." 2>&1
+	MINVER=2.7
+	MINVER3=3.4
+	if [ $(bc <<< "$PYTHON_VERSION < $MINVER || ($PYTHON_VERSION > 3.0 && $PYTHON_VERSION < $MINVER3)") -eq 1 ]; then
```

Comment:
It's just a really old python interface that can't change since it's related to python versions. See: https://docs.python.org/2/library/sys.html#sys.version_info and https://docs.python.org/3/library/sys.html#sys.version_info